frontend/www/js/omegaup/grader/MonacoEditor.vue

```diff
@@ -1,9 +1,21 @@
 <template>
-  <div></div>
+  <div>
+    <label>Font Size: </label>
+    <select
+      ref="fontSizeDropdown"
+      v-model="selectedFontSize"
+      @change="onFontSizeChange"
+    >
+      <option v-for="size in fontSizes" :key="size" :value="size">
+        {{ size }}px
+      </option>
+    </select>
+
+    <div ref="editorContainer" style="width: 100%; height: 100%"></div>
```

Comment:
In omegaUp, we strongly avoid the use of inline styles due to the risks of Cross-Site Scripting (XSS) attacks.

Therefore, it is recommended to use a class that contains the necessary styles.

As you can see, there is already a selector for all the divs that adds the styles you need, so removing the inline styles should work.
```suggestion
    <div ref="editorContainer"></div>
```